lang/java/avro/src/main/java/org/apache/avro/Schema.java

```diff
@@ -1354,9 +1359,14 @@ static Schema parse(JsonNode schema, Names names) {
       } else
         throw new SchemaParseException("Type not supported: "+type);
       Iterator<String> i = schema.getFieldNames();
+
+      Set reserved = SCHEMA_RESERVED;
+      if (type.equals("enum")) {
```

Comment:
The test should instead be 'type == ENUM', no?

Reply:
I just kept it consistent with the other references to it in this file. See line 1274 and 1330. "type" is actually a string here, believe it or not.

Reply:
Sorry, I somehow tagged the wrong line.  I meant line 105, not this one.